stripe/__init__.py

```diff
@@ -45,11 +45,16 @@
 # Webhooks
 from stripe.webhook import Webhook, WebhookSignature  # noqa
 
-from stripe.raw_request import _raw_request as raw_request  # noqa
+from stripe.raw_request import _raw_request  # noqa
 
-from stripe.raw_request import _deserialize as deserialize  # noqa
+raw_request = _raw_request
```

Comment:
Mind adding a comment as to why we are doing this?

Reply:
Can we add a test? Or enable typecheking in some test?

Reply:
Done!

Reply:
(Oh, done for the comment. Re: test, I think that's a good idea, will think about that soon.)